packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -1406,6 +1406,62 @@ You're able to grant third-party admin status when you assign a new role, or you
 
 When this setting is enabled, the admins won't receive any of the default Okta administrator emails. These admins also won't have access to contact Okta Support and open support cases on behalf of your org.
 
+### List Users with Role Assignments
+
+<ApiOperation method="get" url="/api/v1/iam/assignees/users" />
+
+Lists all the Users with Role Assignments
```

Comment:
In this sentence, does "role assignments" refer to an object? If not, we can go lower case.
```suggestion
Lists all the Users with role assignments
```